Makefile

```diff
@@ -4,4 +4,12 @@ PHONY: \
 	build_test_image \
 
 build_test_image:
-	$(DOCKER_COMPOSE) -f docker-compose.yml  run --rm postgres -d
+	docker-compose -f docker-compose.yml  run --rm postgres -d
```

Comment:
I couldn't find in the PR description a justification for that change. Can you expand on that? Also, if we're not using `DOCKER_COMPOSE` anymore, what is the reason to keep it on line 1?

Reply:
No reason, don't know what happened here. Going to rollback this part of the file